src/Error/UserError.php

```diff
@@ -0,0 +1,14 @@
+<?php
+namespace GraphQL\Error;
+
+/**
+ * Class UserError
+ *
+ * Note:
+ * Error that can be display safely to client...
+ *
+ * @package GraphQL\Error
+ */
+class UserError extends InvariantViolation
```

Comment:
I think its an acceptable BC, if we put in release notes, as it is a security enhanchement?